### PR TITLE
show confirmation dialog when leaving chat

### LINKED ISSUE #2378: deleting a chat should show a confirmation dialog
see figma https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=70%3A0

steps
1. right click on a chat, and click 'delete chat'

result: just deletes the chat
expected result: should display a modal to confirm the user really wants to delete this chat

### PR DESCRIPTION
**Pull request by richard-ramos**
Fixes #2378

### CODE REVIEW

**Comment by anastasiyaig** [comment]
@richard-ramos could you please correct the copy for confirmation modal so it says : 

> Are you sure you want to delete this chat? 

Currently is says: **Are you sure you want to this?**

**Review by jrainville — COMMENTED** [review]
Code looks good. Having a better text would be nice though, especially since the one we have has a typo haha.

File: ui/app/AppLayouts/Chat/components/ChannelContextMenu.qml [hunk 1]
```diff
@@ -142,6 +142,7 @@ PopupMenu {
         id: deleteChatConfirmationDialogComponent
         ConfirmationDialog {
             btnType: "warn"
+            confirmationText: qsTr("Are you sure you want to leave this chat?")
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/components/ChannelContextMenu.qml:145]: Should we rename this `channel`? Question for everyone

**Review by jrainville — APPROVED** [review]

**Comment by anastasiyaig** [comment]
thanks for fixing it! As i got we have 1 pop-up confirmation for all types of chat, so **deleting** 1x1 chat leads to confirmation modal asks for **leaving** chat but i think we can address it later



lets merge it

**Pull request merged**